slackbridge/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:wily
+
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
I think we can add some additional variables to the Dockerfile that make it easier for the operator to launch the container. For instance:

```
ENV DEBIAN_FRONTEND noninteractive
    PASSWORD="password" \
    NICKNAME="nickname" \
    CHANNEL="channel-room" \
    TOKEN="token" \
    SERVER="server" \
```